storage/tianmu/core/query.cpp

```diff
@@ -448,9 +448,16 @@ void Query::ExtractOperatorType(Item *&conds, common::Operator &op, bool &negati
       conds = cond_func->arguments()[0];
       break;
     }
-    case Item_func::UNKNOWN_FUNC:
+    case Item_func::UNKNOWN_FUNC: {
       op = common::Operator::O_UNKNOWN_FUNC;
+      if (cond_func->arg_count == 2) {
+        auto subselect_item = dynamic_cast<Item_in_subselect *>(cond_func->arguments()[1]);
```

Comment:
Here, we can use `down_cast`, which implemented by MySQL.

Reply:
Ok, Here we canot sure which type of this args. And, the cast maybe failed. Therefore, keep this.